spec/controllers/application_controller/policy_support_spec.rb

```diff
@@ -0,0 +1,19 @@
+describe ApplicationController do
+  describe "#assign_policies" do
+    let(:admin_user) { FactoryGirl.create(:user, :role => "super_administrator") }
+    let(:host)       { FactoryGirl.create(:host) }
+    before do
+      EvmSpecHelper.create_guid_miq_server_zone
+
+      login_as admin_user
+      allow(User).to receive(:current_user).and_return(admin_user)
+      allow(controller).to receive(:assert_privileges)
+      allow(controller).to receive(:find_checked_items).and_return([host.id])
+    end
+
+    it "testing " do
```

Comment:
This should be a description of what you are testing, here I think something like *redirects to protect* is the right one.